openapi/openapi.yaml

```diff
@@ -1385,6 +1385,41 @@ paths:
               schema:
                 $ref: "#/components/schemas/PythonPackagesCountInfoResponseError"
 
+  /python/package/versions:
+    get:
+      tags: [PythonPackages]
+      x-openapi-router-controller: thoth.user_api.api_v1
+      operationId: list_python_package_versions
+      summary: List versions of the given Python package.
+      parameters:
+        - name: name
```

Comment:
Don't we want to provide a filter by index as well? and what about os_name, os_version, python_interpreter? As PPV for use contain name, version, index, os_name, os_version, python_interpreter?

Reply:
I was first thinking about the same approach. However I'm not sure if that meets desired functionality, this endpoint lists versions of python packages that Thoth knows about. That means all that we monitor, they are not specific to solvers installed. An example of a questionable output can be TensorFlow==2.0.0 for ubi8, py39. As TensorFlow==2.0.0 has no support for Python 3.9, we would not provide any version in the listing, but that's not true in this case.

We can introduce an endpoint that would be specific to solver results (in that way TensorFlow==2.0.0 would not be available in the response). As described in #1112. This endpoint, however, sounds like providing different data than this one.

Reply:
> However I'm not sure if that meets desired functionality, this endpoint lists versions of python packages that Thoth knows about

sounds good, it includes all (name, version, index) no matter if they can be used by adviser